service/test_automatic_run_experiment.py

```diff
@@ -33,7 +33,7 @@
     'experiment': datetime.date(2020, 6, 5),
     'fuzzers': ['honggfuzz', 'afl'],
     'description': 'Test experiment',
-    'oss_fuzz_corpus': True,
+    'oss-fuzz-corpus': True,
```

Comment:
I changed this because a test was failling even on main branch. It was probably introduced in [this PR](https://github.com/google/fuzzbench/pull/1997/files)

Reply:
It seems that the test is still failing, not sure why. This change seemed to fixed it in my local machine 😅

Reply:
No worries.

Reply:
That seems to haunt us since it was pointed out.
I reckon it might be worth fixing if it breaks the CI of all PRs.
Low priority, though.

Reply:
Pull after I land https://github.com/google/fuzzbench/pull/1999 it should fix this problem.

Reply:
Pull now

Reply:
Done